### ISSUE TITLE
[Feature]: towhee可以载入本地的encoder模型吗? 

### ISSUE THREAD

**Issue by micrazy** [towhee-io/towhee#2095]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Is your feature request related to a problem? Please describe.

想用一些中文CLIP模型，但towhee目前似乎不支持自定义模型哦

**Comment by jaelgu** [comment]
@micrazy Towhee supports user model as well. You have two options below:

1. wrap the model inference with a function to work with other towhee operators. Here are some example codes:
```
def encoder(inputs):
    inputs = preprocess(inputs)
    embs = my_model(inputs)
    return embs.detach().cpu().numpy()

towhee.dc(['test']).runas_op(func=encoder).to_list()
```
2. `towhee.models.clip` also supports some multilingual models (support Chinese).  Check README in this page: https://github.com/towhee-io/towhee/tree/main/towhee/models/clip

**Comment by micrazy** [comment]
tks, 我试了下确实可以，不过碰到了另外的问题：

**Comment by jaelgu** [comment]
> tensor_normalize

`tensor_normalize` does normalize the input. You can check docs here: https://towhee.readthedocs.io/en/latest/builtin%20operators/tensor%20like.html . In your case, I think the problem is that you need to squeeze the vec before normalization. i.e. The proper shape should be (512,) instead of (1, 512), which affects the normalization result.

**Comment by micrazy** [comment]
NB!
